POSserver/server/schemas/note.py

```diff
@@ -15,3 +15,24 @@ class Query(ObjectType):
 
     def resolve_all_notes(self, info, **kwargs):
         return Note.objects.all()
+
+class CreateNote(graphene.Mutation):
+    class Arguments:
+        title = graphene.String()
+        content = graphene.String()
+
+    ok = graphene.Boolean()
+    note_field = graphene.Field(Note_Type)
+
+    def mutate(self, info, title, content):
+
+        user = info.context.user
+        if user.is_anonymous:
+            return CreateNote(ok=False, status="Must be logged in.")
+        else:
+            new_note = Note(title=title, content=content, ok=True)
+            new_note.save()
+            return CreateNote(note_field=new_note, ok=True)
+
```

Comment:
There is an auto populated create and modified at field on note. Because they aren’t entered by the user, just populated by the database on save, they aren’t necessary to include as arguments.